packages/api/src/auth/decoders/index.ts

```diff
@@ -4,10 +4,12 @@ import type { SupportedAuthTypes } from '@redwoodjs/auth'
 
 import { netlify } from './netlify'
 import { auth0 } from './auth0'
+import { azureAd } from './azureAd'
 const noop = (token: string) => token
 
 const typesToDecoders: Record<SupportedAuthTypes, Function> = {
   auth0: auth0,
+  azureAd: azureAd,
```

Comment:
Not certain the Java/Typescript naming convention:

* azureAd
*azureAD
* azureActiveDirectory
?

Reply:
@dthyresson Not me either. Personally I would go with `azureAD` but I'm not sure that is the right convention. `azureActiveDirectory` is a bit to long imo. Can we get input from @peterp in this maybe?

Once decided, I'll make that change to all references.

Reply:
Thanks @jeliasson :)

I would personally go with the really long name. Since people who are unfamiliar with it then have a full reference instead of "AD"